### PR TITLE
update answer for switching to a new branch in Git

### PR DESCRIPTION
**Pull request by adnenre**
while it's a new branch that mean it wasn't there 

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Hi @adnenre ! 

Thank you for your contribution. 
A couple of suggestions for your PR.

No.1: 
I think we should also tweak the question. 
We could consider something like this.

How can you create a new branch and switch to it in Git?

No.2:
Whenever you make changes to the quiz answer, you need to make sure to update it in the explanation.
The explanation field is shown to users after each questions and we want to make sure the answers match. 

If you can make those two changes, that would be great! 
Thanks!

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1963,7 +1963,7 @@ const fullQuiz = [
   },
   {
     "Question": "How can you switch to a new branch in Git?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:1965]: I propose making a change to the question too.
How can you create a new branch and switch to it in Git?

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Remember to update the explanation with the new proposed answer

File: src/data/full-quiz.js (hunk 1 above)
Comment (adnenre) [reply src/data/full-quiz.js:1965]: yeah that looks better ;)

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
It looks like the explanation hasn't updated yet with the new answer. On line 1970 of the quiz file, please update the explanation with your new proposed answer. Thanks!

**Review by jdwilkin4 — APPROVED** [review]
LGTM!

**Pull request merged**